gdcdictionary/schemas/core_metadata_collection.yaml

```diff
@@ -462,5 +462,10 @@ properties:
       The identity provider for the data set. Only users who have logged in via this identity provider can request access to this data.
     type: string
 
+  external_url:
+    description: >
+      The external database url for a study.
```

Comment:
Could we change this to "The external download url for a study". My bad, i didn't describe the field in the ticket description